filebeat/crawler/crawler.go

```diff
@@ -44,7 +44,7 @@ func (c *Crawler) Start(r *registrar.Registrar, reloaderConfig *common.Config) e
 	}
 
 	if reloaderConfig.Enabled() {
-		logp.Warn("EXPERIMENTAL feature dynamic configuration reloading is enabled.")
+		logp.Warn("BETA feature dynamic configuration reloading is enabled.")
```

Comment:
I will open a PR to the changelog to add this as known issue in case we can't fix it today.